docs/content/tutorials/new-app/index.md

```diff
@@ -0,0 +1,378 @@
+
+Starting log stream...
+
+demo-bb9df8798-b68rc › demo
+demo-bb9df8798-b68rc demo Using in-memory store: no connection string found
+demo-bb9df8798-b68rc demo Server is running at http://localhost:3000
+demo-bb9df8798-b68rc demo [port-forward] connected from localhost:3000 -> ::3000
+```
+
+Open [http://localhost:3000](http://localhost:3000) to view the Radius demo container:
+
+<img src="demo-landing.png" alt="Screenshot of the Radius demo container" width=500px >
+
+When you're done press `CTRL + c` to terminate the port-forward and log stream.
+
+## Step 5: Add a database and a connection
+
+In addition to containers, you can add dependencies like Redis caches, Dapr State Stores, Mongo databases, and more.
+
+1. Add a Mongo database and an environment parameter to your `app.bicep` file:
+
+   {{% rad file="snippets/2-app-mongo.bicep" embed=true marker="//MONGO" %}}
+
+   {{< alert title="💡 Radius Recipes" color="info" >}}
+   When you added a Mongo database to your application you didn't need to specify _how or where_ to run the underlying infrastructure. The Radius Environment and its Recipes take care of that for you. Just like how the Radius Environment bound your container to a Kubernetes cluster, it also deploys and binds your Mongo database to underlying infrastructure using [Recipes]({{< ref "/guides/recipes/overview" >}}).
+   {{< /alert >}}
+
+1. To learn about the underlying Recipe that will deploy and manage the Mongo infrastructure run [`rad recipe show`]({{< ref rad_recipe_show >}}):
+
+   ```bash
+   rad recipe show default --resource-type Applications.Datastores/mongoDatabases
+   ```
+   
+   You'll see details on the Recipe, including available parameters and defaults:
+
+   ```
+   NAME      TYPE                                    TEMPLATE KIND  TEMPLATE VERSION  TEMPLATE
+   default   Applications.Datastores/mongoDatabases  bicep                            radius.azurecr.io/recipes/local-dev/mongodatabases:latest
+   
+   PARAMETER NAME  TYPE          DEFAULT VALUE   MIN       MAX
+   username        string        admin           -         -
+   password        secureString  Password1234==  -         -
+   database        string                        -         -
+   ```
+
+   If you want to learn more about the Recipe template it's in the [Recipes repo](https://github.com/radius-project/recipes/blob/main/local-dev/mongodatabases.bicep).
+
+1. Add a connection from your container to the Mongo database to tell Radius that your container needs to communicate with the Mongo database:
+
+   {{% rad file="snippets/2-app-mongo.bicep" embed=true marker="//CONTAINER" markdownConfig="{hl_lines=[\"16-20\"]}" %}}
+
+   {{< alert title="💡 Radius Connections" color="info" >}}
+   Radius Connections are more than just bookkeeping. They are used to automatically configure access and connection information for your containers. Learn more in the [containers documentation]({{< ref "/guides/author-apps/containers/overview" >}}).
```

Comment:
Capitalization should be consistent for Container.

Reply:
```suggestion
   Radius Connections are more than just bookkeeping. They are used to automatically configure access for your containers. Learn more in the [containers documentation]({{< ref "/guides/author-apps/containers/overview" >}}).
```